src/modules/inspect-correlation/components/token-viewer/token-viewer.scss

```
@@ -54,3 +54,60 @@
   background: none;
   border: none;
 }
+
+.switch-container {
+  width: 100%;
+  padding: 10px;
+  margin-bottom: -20px;
+  text-align: right
+}
+
+.token-viewer-switch {
+  position: relative;
+  display: inline-block;
+  width: 30px;
+  height: 17px;
+}
+
+/* 
+ * Hide default HTML checkbox
+ */
+.token-viewer-switch input {
+  opacity: 0;
+}
+
+.token-viewer-switch-slider {
```

Can the slider be viewed as a modifier of the standard token-viewer-switch, which would be a standard checkbox?
```suggestion
.token-viewer-switch--slider {
```